src/runtime_src/xocl/api/xlnx/xclEnqueuePeerToPeerCopyBuffer.cpp

```diff
@@ -106,8 +106,8 @@ validOrError(cl_command_queue    command_queue,
   // buffer or subbuffer object and the source and destination regions
   // overlap or if src_buffer and dst_buffer are different sub-buffers
   // of the same associated buffer object and they overlap. The
-  // regions overlap if src_offset ��� dst_offset ��� src_offset + size -
-  // 1, or if dst_offset ��� src_offset ��� dst_offset + size - 1.
+  // regions overlap if src_offset <= dst_offset <= src_offset + size -
```

Comment:
Well, it is not the Unicode solution I would have been expected but at least this works since the 1950's :-)

Reply:
It looks great in all editors